app/components/provider_interface/new_reference_with_feedback_component.rb

```diff
@@ -0,0 +1,116 @@
+module ProviderInterface
+  class NewReferenceWithFeedbackComponent < ViewComponent::Base
+    attr_accessor :reference, :index, :application_choice
+    delegate :feedback,
+             :name,
+             :email_address,
+             :referee_type,
+             :relationship,
+             :relationship_confirmation,
+             :relationship_correction,
+             :safeguarding_concerns,
+             :safeguarding_concerns_status,
+             to: :reference
+
+    def initialize(reference:, index:, application_choice:)
+      @reference = reference
+      @index = index
+      @application_choice = application_choice
+    end
+
+    def rows
+      [
+        name_row,
+        email_address_row,
+        reference_type_row,
+        relationship_row,
+        relationship_confirmation_row,
+        relationship_correction_row,
+        safeguarding_row,
+        safeguarding_concerns_row,
+        feedback_row,
+      ].compact
+    end
+
+  private
+
+    def name_row
+      {
+        key: 'Name',
+        value: name,
+      }
+    end
+
+    def email_address_row
+      {
+        key: 'Email address',
+        value: "#{email_address}#{email_warning}".html_safe,
```

Comment:
Tiny thing but these two should be delimited by newline for prettier HTML

Reply:
after all of this, the warning has been moved! 🎻